.github/workflows/master_auto_merge.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: Auto Merge Double Commit PRs
+on:
+  workflow_call:
+    
+jobs:
+
+  auto-merge:
+    runs-on: non-build
+    env:
+       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
For checkout action I see the latest version is `v4.1.1` ; Can you check if we can use it without any other change/impact. If it fails then open a JIRA and we can upgrade later.

Reply:
Ignore. Create a separate JIRA to upgrade all public actions we are using.

Reply:
https://tessell.atlassian.net/browse/TDEVOPS-1248